nucleus/admin/server-mgmt/src/main/java/com/sun/enterprise/admin/servermgmt/domain/CustomTokenClient.java

```diff
@@ -111,6 +112,9 @@ public Map<String, String> getSubstitutableTokens() throws DomainException {
                             }
                             // Find next available unused port by incrementing the port value by 1
                             while (!NetUtils.isPortFree(port) && !usedPorts.contains(port)) {
+                                if (port > NetUtils.MAX_PORT) {
+                                    throw new DomainException("No free port available or it is prohibited.");
```

Comment:
Seems that other `DomainException`s here use `_strings` to get the message.

Reply:
It just extends RepositoryException which extends java.lang.Exception, no i18n inside. The plan is to get rid all of these messages as they are pretty inconsistent and often copy pasted same to a property file and same "as a default". But that will be for another pr(s).